### ISSUE TITLE
Changing map size causes weird behavior

### ISSUE THREAD

**Issue by phillo3** [KSneijders/AoE2ScenarioParser#42]
Here's a simple code example

```
from AoE2ScenarioParser.scenarios.aoe2_de_scenario import AoE2DEScenario

scenario_folder = "path"

input_path = scenario_folder + "default0.aoe2scenario"
source_scenario = AoE2DEScenario.from_file(input_path)

map_manager = source_scenario.map_manager
unit_manager = source_scenario.unit_manager

print(str(map_manager.map_size))

map_manager.map_size = 240

print(str(map_manager.map_size))

for tile in map_manager.get_square_1d(x1=0, y1=0, x2=10, y2=10):
    print("x = " + str(tile.x) + " y = " + str(tile.y))
```

It spit outs obviously incorrect x y values. get_square_2d seems to behave the exact same.

**Comment by phillo3** [comment]
I was able to find a workaround by by saving and reloading the map after changing the map size.

**Comment by KSneijders** [comment]
Hey, thanks for the report. I'm currently unable to test as I'm not at my pc. 
That being said, can you check what version of  AoE2ScenarioParser you are using? 
This looks like an issue that was fixed in version [`0.1.52` ](https://github.com/KSneijders/AoE2ScenarioParser/blob/master/CHANGELOG.md#0152---2023-march-15).

**Comment by KSneijders** [comment]
@phillo3 Thanks for the report, I was able to easily reproduce the issue thanks to the code provided.
This was indeed a bug and has been fixed! Please update to `0.1.55` using: 

    pip install --no-cache-dir --upgrade AoE2ScenarioParser
